target/linux/lantiq/dts/RS353.dts

```diff
@@ -0,0 +1,318 @@
+/dts-v1/;
+
+/include/ "vr9.dtsi"
+
+/ {
+	model = "RS353 - Bintec Elmeg RS353";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200 init=/etc/preinit";
+		leds {
+                        running = &status;
+                        boot = &status;
+			wifi = &wlan;
+			usb2 = &usb; /* Front port is mapped to device 2 */
+			dsl = &dsl;
+                };
+
+	};
+
+	memory@0 {
+		reg = <0x0 0x8000000>;
+	};
+
+	sram@1F000000 {
+		cgu0: cgu@103000 {
+                        compatible = "lantiq,cgu-xway";
+                        reg = <0x103000 0x1000>;
+                        #clock-cells = <1>;
+			lantiq,phy-clk-src = <0x2>;
+		};
+	};
+
+	fpi@10000000 {
+		localbus@0 {
+			nor-boot@0 {
+				compatible = "lantiq,nor";
+				bank-width = <2>;
+				reg = <0 0x0 0x2000000>;
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				partitions {
+					compatible = "fixed-partitions";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					boardconfig: partition@0 {
```

Comment:
The layout was obtained via the bootloader debug cmds : 

`Configuration Table
-------------------
Board Type:                  201
Identification String:       TELDAT ClosedEyeVisual
Serial Number:               RN7FBA015160002
Logic Address:               0x10020000
Logic Length:                0x20000
Logic Configuration Address: 0x10000000
Logic Configuration Length:  0x20000
Bootmonitor Address:         0x10040000
Bootmonitor Length:          0x20000
Flash Config Adresss:        0x10080000
Flash Config Length:         0x40000
Flash Identification String: CEV
BOSS Header Address:         0x100c0000
BOSS Address in RAM:         0x0
MemAlloc-Buffer:             0x00026820
MemAlloc-Buffer Length:      0x7fc97e0
Receive-Buffer:              0x05fb0000
Receive-Buffer Length:       0x2040000
Flash Size:                  33554432
`